### ISSUE TITLE
CellBoundary does not hav a refType

### ISSUE THREAD

**Issue by ocastrup** [OCXStandard/OCX_Schema#100]
Missing refType on CellBoundary:
```XML
	<xs:element name="CellBoundary" type="ocx:BoundedRef_T">
		<xs:annotation>
			<xs:documentation>The reference to a Panel surface making up one Cell boundary.</xs:documentation>
		</xs:annotation>
	</xs:element>
```

**Comment by ocastrup** [comment]
Add refType=ocx:Panel to CellBoundary:

````XML
	</xs:complexType>
	<xs:element name="CellBoundary" type="ocx:CellBoundary_T">
		<xs:annotation>
			<xs:documentation>The reference to a Panel surface making up one Cell boundary.</xs:documentation>
		</xs:annotation>
	</xs:element>
	<xs:complexType name="CellBoundary_T">
		<xs:complexContent>
			<xs:extension base="ocx:BoundedRef_T">
				<xs:attribute ref="ocx:refType" use="required" fixed="ocx:Panel"/>
			</xs:extension>
		</xs:complexContent>
	</xs:complexType>
```

**Comment by ocastrup** [comment]
Included in #101
